react-stopwatch-component/dist/styles.css

```diff
@@ -0,0 +1,20 @@
+.stopwatch {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.fas {
+  padding: 1em;
+  font-size: 1.5em;
+}
```

Comment:
You don't want to add css rules to existing classes from a css library, this will change its default behavior and might cause issues in other parts of project and will cause confusion with other people working on the project. A better solution is to create your own class and use it where you want to apply the styles:

```CSS
.icon {
  padding: 1em;
  font-size: 1.5em;
}
```